packages/env-loader/src/cli-options.ts

```diff
@@ -78,6 +78,7 @@ export const getCliOptions = (scriptName: string): EnvVars => {
     .option('--DRUPAL_CLIENT_SECRET <secret>', 'Drupal client secret')
     .option('--DRUPAL_PREVIEW_SECRET <secret>', 'Drupal preview secret')
     .option('--DRUPAL_SITE_ID <id>', 'Drupal site ID')
+    .option('--USE_REDIS <boolean>', 'Enable redis')
```

Comment:
Do we want `--USE_REDIS=true` (`--USE_REDIS=false`) or `--USE_REDIS` (`--no-USE_REDIS`)? One actually uses boolean values whereas the other uses string values. I don't love the `no-USE_REDIS` syntax because it mixes kabob and snake case, but that's a product of us using our UPPER_CASE snake formatting directly on the CLI rather than having a conversion. I don't think the conversion is necessary, ultimately, but this case would feel prone to accidently using `--no-USE-REDIS`, perhaps.

Reply:
I think I lean towards booleans here, but would we even need `--no-USE_REDIS`? i would expect `--USE_REDIS` = true, and the absence of that flag = false (same for `--SSG`, if it were bool)

Reply:
Yeah, fair point. I didn't mean to suggest that we'd actually _need_ it, but rather that the utility provides that syntax OOB for boolean values. It probably would only be necessary for a value that is overriding an otherwise true value (from .env file, likely). That is n/a here.